dbt/include/trino/macros/utils/timestamps.sql

```diff
@@ -0,0 +1,12 @@
+{% macro trino__current_timestamp() -%}
+    current_timestamp
+{%- endmacro %}
+
+{% macro trino__snapshot_string_as_time(timestamp) %}
+    {%- set result = "TIMESTAMP '" ~ timestamp ~ "'" -%}
```

Comment:
probably should be lowercase as well to keep it consistent.

Reply:
yeah let's keep it consistent